### PR TITLE
feat(web): full screen view for duplicates

### PR DESCRIPTION
**Pull request by michelheusschen**
Adds a button to view duplicate assets in full screen. Also changed:
- Use relative units instead of pixels
- More descriptive buttons for permanent delete confirmation

Preview where the left button opens the asset viewer
image

Closes #10173

### CODE REVIEW

File: web/src/lib/i18n/en.json [hunk 1]
```diff
@@ -317,6 +317,7 @@
   "archived": "Archived",
   "asset_offline": "Asset offline",
   "assets": "Assets",
+  "assets_moved_to_trash": "Moved {count, plural, one {# asset} other {# assets}} to trash",
```
Comment (danieldietzler) [inline web/src/lib/i18n/en.json:320]: Does `#` work similar to `{count, number}`?

**Review by danieldietzler — APPROVED** [review]
Code looks great to me!

File: web/src/lib/i18n/en.json (hunk 1 above)
Comment (michelheusschen) [reply web/src/lib/i18n/en.json:320]: Yes exactly, we're already using the shorter `#` syntax for some of our existing translations

Comment (danieldietzler) [reply web/src/lib/i18n/en.json:320]: Ah ok, that's cool (and easier)! :D

**Review by alextran1502 — APPROVED** [review]
Thanks so much!

**Pull request merged**